Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 #!groovy
 
-def dockerVersions = ['19.03.13']
+def dockerVersions = ['20.10.0-rc2', '19.03.14']
```

Comment:
We typically don't use RCs here

Reply:
GA should now be available, so I can update. That said; I think it makes sense to test against upcoming releases (once we reach RC's) to make sure they work together.